lib/bitcrowd_ecto/assertions.ex

```diff
@@ -323,18 +323,18 @@ defmodule BitcrowdEcto.Assertions do
       end
   """
   @doc since: "0.1.0"
-  @spec assert_count_difference(Ecto.Repo.t(), module, integer, (() -> any)) ::
+  @spec assert_count_difference(Ecto.Repo.t(), module, integer, (() -> any), keyword) ::
           Changeset.t() | no_return
```

Comment:
I usually specify both functions to make the lib play nicely with dialyzer (if you only specify the `/5`, the `/4` isn't spec'ed - which is ok, as dialyzer will simply assume `any,any,any,any`, but having a proper spec is nicer IMO)

so this would be

```suggestion
  @spec assert_count_difference(Ecto.Repo.t(), module, integer, (() -> any)) ::
          Changeset.t() | no_return
  @spec assert_count_difference(Ecto.Repo.t(), module, integer, (() -> any), keyword) ::
          Changeset.t() | no_return
```